server/src/main/java/no/ntnu/klubbhuset/domain/Order.java

```diff
@@ -0,0 +1,16 @@
+package no.ntnu.klubbhuset.domain;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+@Data
+
+public class Order {
+    @Id
+    String id;
+    String url;
+
+}
```

Comment:
Tror ikke denne brukes. Skal jeg bare slette den?

Reply:
Burn it with fire

Reply:
Den ble brukt så lot den stå